playbooks/roles/certificates/templates/openssl.cnf.j2

```diff
@@ -42,25 +43,37 @@ emailAddress = optional
 
 [ req ]
 distinguished_name = req_distinguished_name
+req_extensions     = req_ext
+
+[ req_ext ]
+subjectAltName = @alt_names
+
+[ alt_names ]
+{% for item in tls_san %}
+IP.{{ loop.index }} = {{ item }}
```

Comment:
Can you also add a `DNS.` SAN here? Even if the value is an IP address I think it will be better for compatibility and these self-signed certs are pretty YOLO already :-)

Reply:
Could we add this later after a first round of testing? It's easy to add later, and it's one fewer dead chicken to swing around if not necessary.

Reply:
Sure. We can take this as a follow-up after a baseline level of compatibility is established. :+1: